domains/transfer/Transfer.API/API.IntegrationTests/API.IntegrationTests.csproj

```diff
@@ -19,7 +19,7 @@
 
     <ItemGroup>
         <PackageReference Include="EnergyOrigin.TokenValidation" Version="3.0.2" />
-        <PackageReference Include="FluentAssertions" Version="6.11.0" />
+        <PackageReference Include="FluentAssertions" Version="6.12.0" />
```

Comment:
Why update?

Reply:
We are using 6.12 in automation worker, so upgraded the transfer as well. We have a task to refine to update our packages